.github/workflows/golangci-lint.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       - name: Lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          version: v1.54.2
+          version: v1.55.2
```

Comment:
This file is synced from `prometheus/prometheus`. We will need https://github.com/prometheus/prometheus/pull/13269 first.

Reply:
Hi @SuperQ, 
#552 can be merged to solve this.

Reply:
Done

Reply:
Good ! Rebase is done now.